modules/4337/contracts/test/Imports.sol

```diff
@@ -0,0 +1 @@
+import "@account-abstraction/contracts/core/EntryPointSimulations.sol";
```

Comment:
nit: For consistency, can we use the same file name as we do for other test dependencies?

In passkeys we use: https://github.com/safe-global/safe-modules/blob/main/modules/passkey/contracts/test/TestDependencies.sol

Also, we can combine this with `SafeContracts.sol` (as those are also just test dependencies that are pulled into the compilation unit for better Hardhat integration, and we don't need two files for that).